examples/oidc/public-dc/react/README.md

```diff
@@ -10,12 +10,14 @@ This is an example react app that authenticates users using OAuth 2.0 standard f
 
 Currently DC is only available in the [Sandbox CSS](https://bcgov.github.io/sso-requests-sandbox) offering, production CSS DC integration coming soon.  
 
-This example specifically uses the verified-email, Verified Credential.  Creating defferent one will require reaching out to the DC team.
+This example specifically uses the verified-email, Verified Credential.  Creating a different one will require reaching out to the DC team.
```

Comment:
Looks great, just noticed one old reference to Verified Credential in this line that could change to Digital Credential

Reply:
oh good catch